package.json

```diff
@@ -31,39 +31,35 @@
     "license-badges": "run-s license-badge license-badge-dev",
     "coverage-badge": "coveradge badges/coverage-badge",
-    "compress-images": "imageoptim \"editor/extensions/*.png\" \"editor/spinbtn/*.png\" \"editor/jgraduate/images/*.{png,gif}\" \"editor/images/*.png\"",
-    "copy": "copyfiles -f node_modules/load-stylesheets/dist/index-es.js editor/external/load-stylesheets && copyfiles -f node_modules/jamilih/dist/jml-es.js editor/external/jamilih && copyfiles -f node_modules/query-result/esm/index.js editor/external/query-result/esm && copyfiles -f node_modules/qr-manipulation/dist/index-es.js editor/external/qr-manipulation/dist && copyfiles -f node_modules/stackblur-canvas/dist/stackblur-es.js editor/external/stackblur-canvas/dist && copyfiles -f node_modules/regenerator-runtime/runtime.js editor/external/regenerator-runtime && copyfiles -f \"node_modules/core-js-bundle//minified*\" editor/external/core-js-bundle && copyfiles -f \"node_modules/underscore/underscore-*\" editor/jspdf && copyfiles -f \"node_modules/deparam/dist/deparam.esm.js\" editor/external/deparam",
+    "compress-images": "imageoptim \"src/editor/extensions/*.png\" \"src/editor/spinbtn/*.png\" \"src/editor/jgraduate/images/*.{png,gif}\" \"src/editor/images/*.png\"",
+    "copy": "copyfiles -f node_modules/load-stylesheets/dist/index-es.js src/external/load-stylesheets && copyfiles -f node_modules/jamilih/dist/jml-es.js editor/external/jamilih && copyfiles -f node_modules/query-result/esm/index.js editor/external/query-result/esm && copyfiles -f node_modules/qr-manipulation/dist/index-es.js editor/external/qr-manipulation/dist && copyfiles -f node_modules/stackblur-canvas/dist/stackblur-es.js editor/external/stackblur-canvas/dist && copyfiles -f node_modules/regenerator-runtime/runtime.js editor/external/regenerator-runtime && copyfiles -f \"node_modules/core-js-bundle//minified*\" editor/external/core-js-bundle && copyfiles -f \"node_modules/underscore/underscore-*\" editor/jspdf && copyfiles -f \"node_modules/deparam/dist/deparam.esm.js\" editor/external/deparam",
     "remark": "remark -q -f .",
     "build-html": "babel-node --plugins @babel/plugin-transform-modules-commonjs build/build-html.js",
-    "eslint-fix": "eslint --cache --fix --ext js,md,html .",
-    "eslint": "eslint --max-warnings 0 --cache --ext js,md,html .",
+    "eslint-fix": "eslint --fix --ext js,md,html src",
+    "eslint": "eslint --ext js,html src",
     "rollup": "rollup -c",
     "build-by-config": "rollup -c rollup-config.config.js",
     "prep-no-core-rollup": "run-s eslint build-html build-by-config",
     "prep": "run-s prep-no-core-rollup rollup",
     "prepublishOnly": "run-s license-badges copy prep build-docs",
     "cypress:open-no-start": "cypress open",
+    "cypress:instrument": "rimraf instrumented && cp -r src instrumented && nyc instrument --compact false src instrumented",
```

Comment:
It was easier for me to remove the linter on the documentation but I'm OK to put it back.

Regarding copyfiles, I tested it didn't work on my environment (macOS). I left it unchanged but I will try to find the issue.

Reply:
I see that you decided to use cp-cli instead. We will have to replace copyfiles by cp-cli in other instances.

Reply:
`copyfiles` was having some issues with directories--it was not copying all files for some reason and there seemed to be no options to work properly (seems it may have been an issue with the underlying globbing library). Yes, sure. There are Rollup plugins like https://github.com/bengsfort/rollup-plugin-copy-assets or https://github.com/vladshcherbin/rollup-plugin-copy that might help.